odilia/src/events/object.rs

```diff
@@ -480,15 +480,15 @@ mod state_changed {
 		let state_value = event.enabled == 1;
 		// update cache with state of item
 		let a11y_prim = AccessiblePrimitive::from_event(event)?;
-		match update_state(state, &a11y_prim, event.state, state_value) {
-			Ok(false) => tracing::error!("Updating of the state was not succesful! The item with id {:?} was not found in the cache.", a11y_prim.id),
-			Ok(true) => tracing::trace!("Updated the state of accessible with ID {:?}, and state {:?} to {state_value}.", a11y_prim.id, event.state),
-			Err(e) => return Err(e),
-		};
+		if update_state(state, &a11y_prim, event.state, state_value)? {
+			tracing::debug!("Updating of the state was not succesful! The item with id {:?} was not found in the cache.", a11y_prim.id);
```

Comment:
debug is more acceptable, but also the spelling error remains. One can always lower it to trace as well, because cache is initially not consistent anyway, and even debug log can be clogged by this, but since it's not error anymore, this is just a suggestion